datasets/src/lib.rs

```diff
@@ -121,6 +127,33 @@ pub fn winequality() -> Dataset<f64, usize> {
         .with_feature_names(feature_names)
 }
 
+#[cfg(feature = "linnerud")]
+/// Read in the physical exercise dataset from dataset path.
+///
+/// Linnerud dataset contains 20 samples collected from 20 middle-aged men in a fitness club.
```

Comment:
this is the smallest dataset I ever encountered :laughing:

Reply:
> but according to Github your archive is empty

I made the gz archives on Windows :grin:. I am going to use a Unix box...
> Can you re-order the struct in src/iter.rs

Ok